entgql/internal/todoplugin/ent/schema/durationgql/durationgql.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019-present Facebook
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package durationgql
```

Comment:
ditto

Reply:
Can't you import it from internal/todo/schema/durationgql?

Reply:
Let's me try, so I can test the mapping config for external types.